assignment/index.html

```diff
@@ -64,9 +65,24 @@
 
       var jsonToCsv = function(json) { console.log(json); };
 
+      //Log series of arrays
+      var getValues = (x) => {return Object.values(x)}
+      var allHealthCenters = healthCenters.map(getValues)
+      var getKeys = [Object.keys(healthCenters[0])]
+      console.log(getKeys.concat(allHealthCenters))
 
       var addMarkers = function(map) {};
 
+      // Function to create markers and add them to the map
+      // Only health centers with ZIP codes between 1940 and 1949
+      for (i=0; i<healthCenters.length; i=i+1){
+        if (healthCenters[i].ZIP>=19140 & healthCenters[i].ZIP<=19149) {
+          marker = L.marker([healthCenters[i].LAT,healthCenters[i].LNG]);
+          marker.bindPopup(healthCenters[i].Name);
```

Comment:
![image](https://user-images.githubusercontent.com/8103418/52172399-60f99080-273c-11e9-8959-1bb2da15101d.png)

Looks like the popup name isn't working properly